homeassistant/components/luxtronik/__init__.py

```diff
@@ -0,0 +1,157 @@
+"""Support for Luxtronik heatpump controllers."""
+# region Imports
+from luxtronik import LOGGER as LuxLogger
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST, CONF_PORT, EVENT_HOMEASSISTANT_STOP
+from homeassistant.core import Event, HomeAssistant
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.typing import ConfigType
+
+from .const import (
+    CONF_LANGUAGE_SENSOR_NAMES,
+    CONF_LOCK_TIMEOUT,
+    CONF_SAFE,
+    DOMAIN,
+    LANG_DEFAULT,
+    LOGGER,
+    LUX_SENSOR_DETECT_COOLING,
+    PLATFORMS,
+)
+from .helpers.helper import get_sensor_text
+from .helpers.lux_helper import get_manufacturer_by_model
+from .luxtronik_device import LuxtronikDevice
+
+# endregion Imports
+
+# region Constants
+LuxLogger.setLevel(level="WARNING")
+# endregion Constants
+
+
+async def async_setup_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> bool:
+    """Set up from config entry."""
+    hass.data.setdefault(DOMAIN, {})
+
+    LOGGER.info(
```

Comment:
I don't think we need to log this?
And if it is really needed, we should make it debug level.

Reply:
Changed to debug